daemons/based/based_io.c

```diff
@@ -50,7 +50,7 @@ cib_rename(const char *old)
     new_fd = mkstemp(new);
     crm_err("Archiving unusable file %s as %s", old, new);
     if ((new_fd < 0) || (rename(old, new) < 0)) {
-        crm_perror(LOG_ERR, "Couldn't rename %s as %s", old, new);
+        crm_err("Couldn't rename %s as %s: %s", old, new, pcmk_rc_str(errno));
```

Comment:
This has an additional problem -- the first crm_err() could theoretically change errno.

We could save errno right after mkstemp(), but I think it doesn't make sense to log an "Archiving" error then a "Couldn't rename" error -- let's just do `if ... "Couldn't archive unusable file %s (disabling disk writes and continuing)" ... else ... "Archived unusable file ..."`

Reply:
I thought that one was pretty weird, too.  Fixed.